Extension/UqlFunctionCaller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Extension;
+
+use Netdudes\DataSourceryBundle\Extension\Exception\FunctionNotFoundException;
+
+class UqlFunctionCaller
```

Comment:
seems a bit of a superfluous class for just calling something else? is this the best way?